### PR TITLE
<Input/> - added `warning` status

### PR DESCRIPTION
**Pull request by malachico**
### 🔦 Summary
...

### ✅ Checklist
- [ ] 👨‍💻 API change is approved by the UI Infra Developers 
- [ ] 👨‍🎨 UX change is approved by the Design System UX 
- 📚 Change is documented
  - [ ] Story
  - [ ] API description
  - [ ] Other (explain)
- 🔬 Change is tested
  - [ ] Component
  - [ ] Visual test

added warning state to input component. 
image

### CODE REVIEW

**Comment by NitayRabi** [comment]
- Regarding code - is AmaterialTheme still needed - @shlomitc ? beyond that seems good, in general (not for this scope) we should use `Input.Affix` compositions to achieve this behaviours (both for warning, and error).
- Regarding UX - you based the design on [this](https://zpl.io/2pBr6Lj) spec? was the colors for hover, focus approved by design system? @LittleTal @milkyfruit

**Comment by shlomitc** [comment]
Amaterial theme not needed
looks like it's not matching the UX specs, the icon should definitely be different
https://zpl.io/2pBr6Lj
image

File: src/Input/Input.js [hunk 1]
```diff
@@ -469,7 +470,11 @@ Input.propTypes = {
   disabled: PropTypes.bool,
 
   /** Input status - use to display an status indication for the user. for example: 'error' or 'loading' */
-  status: PropTypes.oneOf([Input.StatusError, Input.StatusLoading]),
+  status: PropTypes.oneOf([
```
Comment (NitayRabi) [inline src/Input/Input.js]: Please update the comment to reflect the `warning` option in the docs.

File: src/Input/ThemedInputWarningSuffix.js [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+import styles from './Input.scss';
+
+const placementToMoveBy = {
+  right: { x: 2, y: -10 },
+  left: { x: -2, y: -10 },
+  top: { x: 0, y: -10 },
+  bottom: { x: 0, y: -8 },
+};
+
+const AmaterialWarningSuffix = ({
```
Comment (NitayRabi) [inline src/Input/ThemedInputWarningSuffix.js]: `AmaterialWarningSuffix` is no longer relevant

**Comment by LittleTal** [comment]
As @shlomitc mentioned, the warning icon is not aligned:

image

Should be:

image

**Comment by malachico** [comment]
hi @LittleTal and @shlomitc thanks a lot for the review. 
I couldn't find a triangle alert icon,
as I understood from Jasmin Adam, it should be added to WSR, and then we will switch the circle to triangle.
can we use for now with the circle one until the triangle will be added and then replace it? I don't want this to delay our development. 
thanks.

**Comment by LittleTal** [comment]
Here is the icon: 
https://wix-wix-ui-icons-common.surge.sh/?selectedKind=Icons%2FSystem&selectedStory=Normal&full=0&addons=0&stories=1&panelRight=0

**Comment by malachico** [comment]
Hi, thanks a lot for the reviews, I've fixed your comments :)
image

**Pull request merged**